recipe/patches/0026-Unvendor-expat.patch

```diff
@@ -0,0 +1,204 @@
+From 694404bc76fae025c40894c6d727ec6b70a95f84 Mon Sep 17 00:00:00 2001
+From: Thomas Bininda <schribl@gmail.com>
```

Comment:
Can you keep my credits here?
You can add yourself as a `co-authored-by` if you changed the patch.

Reply:
Did update it, sorry I did not want to take any credit I just did the same changes as in the patch and created a patch with git as the existing patch for other versions did not apply cleanly.

I checked the other versions and there is no co-author so I also did not add one.